tests/virt/cluster/longevity_tests/test_multi_vm_upgrade_and_reboot.py

```diff
@@ -3,10 +3,7 @@
 from tests.virt.cluster.longevity_tests.constants import WSL2_DV_PARAMS, WSL2_VM_PARAMS
 from tests.virt.cluster.longevity_tests.utils import run_windows_upgrade_storm
 
-pytestmark = [
-    pytest.mark.usefixtures("skip_if_workers_vms", "skip_test_if_no_ocs_sc"),
-    pytest.mark.longevity,
-]
+pytestmark = [pytest.mark.usefixtures("skip_test_if_no_ocs_sc"), pytest.mark.longevity, pytest.mark.special_infra]
```

Comment:
wait, why do you need OCS here? Can't this test run on any other storage?

Reply:
I think we can remove this skip
we are going to have special cluster for all our tests, I would expect to have ocs as part of cluster for all runs..

how about adding it to `virt_special_infra_sanity` verification?

Reply:
that's intentional. this pr was getting too big so i decided to split it. the cpu and storage related skips will be handled in separate one. i'll add this to pr description

Reply:
@dshchedr there are plans to use a completely different storage in the nearest future (for all jobs), so we should not rely on the storage class name, but we need to understand which storage capability is required for this test. (not necessarily in this PR)